GooglePlayInstant/Tests/Editor/QuickDeploy/LoadingBarGeneratorTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using GooglePlayInstant.Editor.QuickDeploy;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using System;
+using Object = UnityEngine.Object;
```

Comment:
Can you please do Code -> Optimize Imports ? Looks like there may be some extras here.